src/sentry/services/hybrid_cloud/identity/impl.py

```diff
@@ -37,18 +37,31 @@ def get_provider(
     def get_identity(
         self,
         *,
-        provider_id: int,
         user_id: int | None = None,
         identity_ext_id: str | None = None,
+        provider_id: int | None = None,
+        provider_ext_id: str | None = None,
+        provider_type: str | None = None,
```

Comment:
Needed to extend this service method for some of the webhook changes.

Reply:
Should we be moving these parameters to a TypedDict and using the `filter` parameter instead?